README.md

```diff
@@ -28,7 +28,7 @@ most important part. Add your answer to this markdown file.
 
 The base cases would be if the array has 0 elements, or the array has one element. The constants would include calculations and adding the final results. 
 
-Since we were required to split the array into 3 subarrays, the time would be equivalent to $3T(n / 3)$. This is because each time the input is divided into thirds and there are 3 calls being made recursively. Knowing this, $T(n) = 3T(n / 3)$ when n > 1, and if n <= 1 it should be T(n) = 1. We need to substitute n/3 for each n so the previous statement will turn into 9T(n/9). Therefore, in order to get the base case, $T(n) = (3^i) T(n/3^i)$ where i is going to be the number of recursive steps. Since we have this pattern in order to get the base case, we must have 3^logbase3(n) * T(n/3^(logbase3(n))) where n*T(1) results in n. Also, since the sum grows linearly, then we can find the toal time complexity will be $\Theta(n)$.
+Since we were required to split the array into 3 subarrays, the time would be equivalent to $3T(n / 3) + O(1)$. This is because each time the input is divided into thirds and there are 3 calls being made recursively, the $O(1)$ represents the work done outside of the recursive calls. Knowing this, $T(n) = 3T(n / 3) + O(1)$ when n > 1, and if n <= 1 it should be T(n) = 1. We need to substitute n/3 for each n so the previous statement will turn into 9T(n/9). Therefore, in order to get the base case, $T(n) = (3^i) T(n/3^i)$ where i is going to be the number of recursive steps. Since we have this pattern in order to get the base case, we must have 3^logbase3(n) * T(n/3^(logbase3(n))) where n*T(1) results in n. Also, since the sum grows linearly, then we can find the toal time complexity will be $\Theta(n)$.
```

Comment:
For the recurrence relation, you can't use O -- please quantify the amount of work done exactly.